README.md

```diff
@@ -99,6 +99,8 @@ Then when running ESLint add `--ext '.js,.ts'` (you might need also `.jsx, .tsx`
 yarn eslint --ext '.js,.ts' ./src
 ```
 
+`parserOptions.project` is set to `./tsconfig.json`. You may need to adjust that. [Read more](https://typescript-eslint.io/architecture/parser#project)
```

Comment:
```suggestion
`parserOptions.project` is set to `./tsconfig.json`. You may need to [adjust that](https://typescript-eslint.io/architecture/parser#project).

To do so, you'll need to override our setup for TS files in your ESLint config:
```js
overrides: [
  {
    files: ['*.ts', '*.tsx'],
    parser: '@typescript-eslint/parser',
    plugins: ['@typescript-eslint/eslint-plugin'],
    parserOptions: {
      project: './tsconfig.json'
    }
  }
]
```
```

Reply:
I think this would be helpful. Please verify if the advice is sound and works as intended 🙏🏼

Reply:
Since the default is `./tsconfig.json` I suggest to have the value be `./packages/**/tsconfig.json` in the README as I assume it would be the most common case